onnx2torch/node_converters/pad.py

```diff
@@ -64,8 +74,21 @@ def __init__(
         self.pads = pads
         self.constant_value = constant_value
 
+    @classmethod
+    def create_from_onnx_params(  # pylint: disable=missing-function-docstring
```

Comment:
You use this method as constructor of the class everywhere. Why don't you replace the __init__ method with this method?

Reply:
But you can explicitly convert onnx attributes to torch attributes and use standart __init__

Reply:
But I need apply this code in 2 places (the same code)